test/e2e/driver/driver.go

```diff
@@ -84,8 +84,8 @@ func getStorageClass(
 	}
 }
 
-func getVolumeSnapshotClass(generateName string, provisioner string) *v1alpha1.VolumeSnapshotClass {
```

Comment:
Hi @xing-yang I am trying to upgrade v1alpha1 to v1beta1, and always hit following error when running e2e test in creating snapshot class(test code is [here](https://github.com/andyzhangx/azuredisk-csi-driver/blob/0b119a046385dc2aa6474768637c2d965159fb76/test/e2e/testsuites/testsuites.go#L106)), while if I create snapshot by `kubectl` manually, it works well.
```
    Unexpected error:
        <*errors.StatusError | 0xc00043ec80>: {
            ErrStatus: {
                TypeMeta: {Kind: "", APIVersion: ""},
                ListMeta: {
                    SelfLink: "",
                    ResourceVersion: "",
                    Continue: "",
                    RemainingItemCount: nil,
                },
                Status: "Failure",
                Message: "the server rejected our request due to an error in our request (post volumesnapshotclasses.snapshot.storage.k8s.io)",
                Reason: "Invalid",
                Details: {
                    Name: "",
                    Group: "snapshot.storage.k8s.io",
                    Kind: "volumesnapshotclasses",
                    UID: "",
                    Causes: [
                        {
                            Type: "UnexpectedServerResponse",
                            Message: "unknown",
                            Field: "",
                        },
                    ],
                    RetryAfterSeconds: 0,
                },
                Code: 422,
            },
        }
```

Reply:
do you know what's the possible reason? My k8s lib of csi driver is based on 1.15, is that the root cause, shall I upgrade to 1.17. 
Current situation is only snapshot e2e test failed on creating snapshot class, while manual snapshot e2e test works by using `kubectl`  
@xing-yang Thanks

Reply:
What is the Kubernetes version you are running?  Do you have alpha version of CRDs installed on the test bed before?

Reply:
thanks, with help of @xing-yang this failure is due to `VolumeSnapshotClass.DeletionPolicy` is required in v1beta1 while it's optional in v1alpha1